src/test/scala/org/ergoplatform/utils/ErgoGenerators.scala

```diff
@@ -73,6 +73,19 @@ trait ErgoGenerators extends CoreGenerators with Matchers {
     x
   }
 
+  def kvGen(keySize: Int, valuesSize: Int): Gen[(Array[Byte], Array[Byte])] = for {
+    key <- genBytes(keySize)
+    value <- genBytes(valuesSize)
+  } yield (key, value)
+
+  lazy val extensionGen: Gen[Extension] = for {
+    headerId <- modifierIdGen
+    mandatoryElements <- Gen.nonEmptyListOf(kvGen(Extension.MandatoryFieldKeySize, 6).map(kv => (kv._1, kv._2)))
```

Comment:
Looks like we don't need ```.map(kv => (kv._1, kv._2))``` here as `kv` is already a tuple of two elements. 

Same for the next line